src/NServiceBus.Persistence.AzureTable/Subscriptions/SubscriptionStorage.cs

```diff
@@ -29,6 +28,16 @@ protected override void Setup(FeatureConfigurationContext context)
 
             var subscriptionTableName = context.Settings.GetSubscriptionTableName();
             var cacheFor = context.Settings.GetOrDefault<TimeSpan>(WellKnownConfigurationKeys.SubscriptionStorageCacheFor);
+
+            context.Settings.AddStartupDiagnosticsSection(
+                "NServiceBus.Persistence.AzureTable.Subscriptions",
+                new
+                {
+                    ConnectionMechanism = cloudTableClientProvider is CloudTableClientForSubscriptionsFromConnectionString ? "ConnectionString" : "Custom",
+                    TableName = subscriptionTableName,
+                    CacheFor = cacheFor,
```

Comment:
Should be outputting “default” if that’s used. Also, no matter what we do, writing all TimeSpan fields brings no value to customers.

Reply:
Switched to CloudTableClient instead